.env.example

```
@@ -0,0 +1,41 @@
+
+# Configure these with your own Docker registry images
+
+DOCKER_IMAGE_BACKEND=kaapi-guardrails-backend
+
+# Callback Timeouts (in seconds)
+CALLBACK_CONNECT_TIMEOUT=3
+CALLBACK_READ_TIMEOUT=10
+
+# require as a env if you want to use doc transformation
+OPENAI_API_KEY="<ADD-KEY>"
+GUARDRAILS_HUB_API_KEY="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJnaXRodWJ8MjcwNzk1NTgiLCJhcGlLZXlJZCI6ImYxZDlkMDlhLTliMDYtNDg0OC1iNjI4LTQ3ZDk3NjA0ZTJiZCIsInNjb3BlIjoicmVhZDpwYWNrYWdlcyIsInBlcm1pc3Npb25zIjpbXSwiaWF0IjoxNzY0NzQ3NDMxLCJleHAiOjE3NzI1MjM0MzF9.DfZq03eG2ClBcIcp5mH5umYxzjeqp2Fp0yPq1WSHBPo"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)